### PR TITLE
feature[common]: added button yes/no

### PR DESCRIPTION
**Pull request by Beymar-Cejas**

### CODE REVIEW

File: src/ui/atoms/ButtonYesNo/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { IProps } from './types/IProps';
+import styles from './styles.module.scss';
+
+function Button({ children, variant = 'primary' }: IProps) {
+  const colorFont = styles[`button-label--${variant}`];
```
Comment (Ditmar) [inline src/ui/atoms/ButtonYesNo/Button.tsx:5]: You should use another approach to set up your variants, check this example to fix your pr . https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7-L10

**Review by Ditmar — CHANGES_REQUESTED** [review]
please rebase your pr, and check the comments

**Comment by Beymar-Cejas** [comment]
https://github.com/Ditmar/backoffice-openscience/tree/feature/SEM-038/src/ui/components/atoms/ButtonYesNo

**Comment by DanX3N0** [comment]
post the link of the storybook component

**Comment by Beymar-Cejas** [comment]
this is storybook
https://github.com/Ditmar/backoffice-openscience/blob/455fe03a4119d585913bbcb20a27ed5aa9c415ed/src/ui/components/atoms/ButtonYesNo/Button.stories.tsx

**Comment by Beymar-Cejas** [comment]
feature/SEM-038
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-038/?path=src/ui/atoms/ButtonYesNo

**Comment by DanX3N0** [comment]
remove the shadow: 
{97CE8C94-5153-4A58-AC29-95DFF6F89F64}

**Review by Ditmar — APPROVED** [review]

**Pull request merged**